toolchain/driver/compile_subcommand.cpp

```diff
@@ -446,6 +472,13 @@ class CompilationUnit {
   // Lower SemIR to LLVM IR.
   auto RunLower() -> void;
 
+  // Runs the optimization pipeline.
+  auto RunOpt() -> void;
+
+  // Runs post-lowering-to-LLVM-IR logic. This is always called if we do any
+  // lowering work, after we've finished building the IR.
```

Comment:
I'm sure this is very explicit but as a less familiar person, I think it could be even more.

```suggestion
  // Runs post-lowering-to-LLVM-IR logic. This is always called if we do any
  // lowering work, after we've finished building the IR in RunLower() and,
  // optionally, RunOpt().
```